README.md

```diff
@@ -1,15 +1,16 @@
 # RIF Relaying Services SDK sample dApp
 
-This a basic sample dApp to showcase how users can submit transactions to the RSK blockchain paying a RelayServer for the transaction fees with tokens. You will need to connect to the dApp with Metamask but only for signing transactions with the account that controls your Smart Wallets.
+This a basic sample dApp to showcase how users can submit transactions to the RSK blockchain paying a RelayServer for the transaction fees with tokens using RIF Relay SDK. You will need to connect to the dApp with Metamask but only for signing transactions with the account that controls your Smart Wallets.
 
-See https://github.com/rsksmart/enveloping for the RIF Enveloping project.
+See https://github.com/rsksmart/rif-relay for the RIF Relay project.
 
 ## Using the dApp
 
 ## Pre-Requisites
```

Comment:
it'd be nice to specify which node version we're using

Reply:
@wilsoniovlabs please correct me if i'm wrong, but this still has not been provided